swift-sdk/Resources/IterableEmbeddedView.xib

```diff
@@ -66,7 +66,7 @@ Description text</string>
                                     <rect key="frame" x="268" y="0.0" width="100" height="100"/>
                                     <constraints>
                                         <constraint firstAttribute="height" constant="100" id="8cm-im-EKi"/>
-                                        <constraint firstAttribute="width" constant="100" id="bVg-Ev-iLm"/>
+                                        <constraint firstAttribute="width" relation="lessThanOrEqual" constant="100" id="bVg-Ev-iLm"/>
```

Comment:
This is now the default behavior allowing Label and text to have more horizontal space when image is not available.